tests/test_automatic_fields.py

```diff
@@ -59,8 +57,10 @@ def test_run(self):
 
         # Run a sync job using orchestrator
         record_count_by_stream = self.run_and_verify_sync(conn_id)
+        synced_records = runner.get_records_from_target_output()
 
         actual_fields_by_stream = runner.examine_target_output_for_fields()
+        stream_primary_keys = self.expected_primary_keys()
 
         for stream in self.expected_streams().difference(untested_streams):
             with self.subTest(stream=stream):
```

Comment:
We have skipped the primary keys assertion as for each streams in the sync mode, the tap first fetches the actual stream records and any updates via the events endpoint which also retrieves the created as well as updated records. Hence there will be 2 records of the same primary key.